downloads/tests/test_download.py

```diff
@@ -0,0 +1,91 @@
+import os.path
+
+from django.test import TestCase
+from django.test.client import Client
+from django.contrib.auth.models import User
+from django.core.files import File as DjangoFile
+from django.core.urlresolvers import reverse
+from django.conf import settings
+from django.contrib.sites.models import Site
+
+from downloads.models import Download
+from downloads.signals import download_requested
+
+RES_DIR = os.path.join(os.path.dirname(__file__), 'res')
+FILE_PATH = os.path.join(RES_DIR, 'test_file.py')
+
+
+class DownloadsTestCase(TestCase):
+
+    def setUp(self):
+        self.username = 'username'
+        self.password = 'password'
+        self.user = User.objects.create_user(
+            self.username, 'user@host.com', self.password
+        )
+        self.client = Client()
```

Comment:
Call the `super` method and then you won't have to redeclare client. By overriding the setUp method and not calling super you're clobberring the work done by super.